### PR TITLE
Proposal for new first section in introduction to fix #144

### LINKED ISSUE #144: first section in introduction doesn't mention the standard
I feel the introduction jumps too fast to explaining public code without introducing the standard, it's purpose, or the foundation.

The text under the last header `goals` gives this context, but only at the end.

### PR DESCRIPTION
**Pull request by clausmullie**
fixes #144

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -1,11 +1,19 @@
 # Introduction
 
-Public code is both civil code (like policy or regulation) and computer source code (such as software and algorithms) executed in a public context, by humans or machines.
+The Standard for Public Code is a set of criteria that supports public organisations in developing and maintaining software together.
```
Comment (bvhme) [inline introduction.md]: This needs a mention of policy as well (especially since you start with that the next sentence)

File: introduction.md [hunk 2]
```diff
@@ -1,11 +1,19 @@
 # Introduction
 
-Public code is both civil code (like policy or regulation) and computer source code (such as software and algorithms) executed in a public context, by humans or machines.
+The Standard for Public Code is a set of criteria that supports public organisations in developing and maintaining software together. 
 
-The Foundation for Public Code makes the explicit distinction between regular software and public software because it operates under fundamentally different circumstances and expectations.
+Anyone developing software or policy for a public purpose can use this standard to work towards higher quality public services that are more cost effective, with less risk and more local control.
```
Comment (bvhme) [inline introduction.md]: I don't know if the qualifier *local* is adding that much here.

File: introduction.md [hunk 3]
```diff
@@ -1,11 +1,19 @@
...
+
+This introduction introduces the term public code, explains why this is important, and introduces the process through which software and policy code can become certified public code. 
+
+The Standard for Public Code is maintained by the Foundation for Public Code, and we'd love to hear from you on how we could improve it!
 
 ## What is public code
 
+Public code is both civil code (like policy or regulation) and computer source code (such as software and algorithms) executed in a public context, by humans or machines. Public code is explicitly distinct from regular software because it operates under fundamentally different circumstances and expectations.
```
Comment (bvhme) [inline introduction.md]: *Civic* code, not *civil* code.

File: introduction.md [hunk 4]
```diff
@@ -28,7 +36,7 @@ In the last decade, public organisations that purchased complete software soluti
 
 ### Technological sovereignty and democratic accountability
 
-Public institutions, civil servants and residents deserve better.
+Public institutions, civic servants and residents deserve better.
```
Comment (ElenaFdR) [inline introduction.md]: This looks like a search and replace error.
```suggestion
Public institutions, civil servants and residents deserve better.
```

**Pull request merged**